client/ayon_substancepainter/api/lib.py

```diff
@@ -435,6 +447,11 @@ def get_parsed_export_maps(config, strip_texture_set=False):
     for key, filepaths in outputs.items():
         texture_set, stack = key
 
+        texture_name = (
+            substance_painter.textureset.TextureSet.from_name(texture_set)
+        )
+        tile_names = set(tile.name for tile in texture_name.all_uv_tiles())
```

Comment:
This is so confusing....

The first variable here `texture_set` apparently is actually the texture set name.
Then using that argument you query the actual texture set object, and you name it `texture_name`.

Which means at that point `texture_set` and `texture_name` are actually opposite/reversed.

I'd refactor:
- `texture_set` -> `texture_set_name`
- `texture_name` - > `texture_set`